vumi/application/tests/test_sandbox.py

```diff
@@ -400,6 +400,11 @@ def test_sandbox_command_does_not_parse_timestamps(self):
 
 class JsSandboxTestMixin(object):
 
+    BIGGER_RLIMITS = {
+        "RLIMIT_STACK": [2 * 1024 * 1024] * 2,
+        "RLIMIT_AS": [256 * 1024 * 1024] * 2,
+    }
```

Comment:
When we run these tests on pypy, node.js fails to start with the default (smaller) rlimits. My guess is that pypy has a larger memory footprint than CPython and the default rlimits are borderline for node.js+CPython already.